workers_launch_template.tf

```
@@ -351,6 +351,43 @@ resource "aws_launch_template" "workers_launch_template" {
     }
   }
 
+  dynamic "block_device_mappings" {
+    for_each = lookup(var.worker_groups_launch_template[count.index], "additional_volumes", lookup(local.workers_group_defaults, "additional_volumes", []))
```

```suggestion
    for_each = lookup(var.worker_groups_launch_template[count.index], "additional_volumes", local.workers_group_defaults["additional_volumes"])
```

for consistency and `local.workers_group_defaults["additional_volumes"]` is already set by default to `[]`